R/modelFitting.R

```diff
@@ -70,12 +70,13 @@ compBayes <- function(dat,
                      nCores = 1,
                      iter = 2000,
                      normalize = TRUE,
+                     normalize_ptm = FALSE,
                      pop_sd = 10,
                      simpleMod = FALSE,
                      bridge = TRUE,
                      adapt_delta = .9,
-                     varPool = 1
-                     ){
+                     varPool = 1,
+                     no_bio = FALSE                     ){
```

Comment:
I prefer that flags have the affirmative sense as their default. I would prefer `is_bio` with a default of `TRUE` rather than `no_bio` with a default of `false`